pkg/istio-agent/sds-agent.go

```diff
@@ -358,7 +358,7 @@ func (sa *Agent) Start(isSidecar bool, podNamespace string) (*sds.Server, error)
 
 	// TODO: remove the caching, workload has a single cert
 	if sa.WorkloadSecrets == nil {
-		workloadSecretCache, _ := sa.newSecretCache(serverOptions)
+		workloadSecretCache, _ := sa.newSecretCache(serverOptions, false)
```

Comment:
Please don't add a new method parameter if you have options. We have enough methods with large number of hard to understand params.

Reply:
Also when you add it to options ( preferably after the cleanup PR is merged ) - that's the right place to add comments on what the option does and use a proper name. isRotate doesn't tell me much. Is rotate what ?

Reply:
Will remove this flag in newSecretCache function